lessons/ro/chapter_6.yaml

```
@@ -129,25 +148,25 @@
 
     * `to_lowercase`/`to_uppercase` pentru a schimba între majuscule și minuscule.
 
-    * `trim` pentru eliminarea spatiilor.
+    * `trim` pentru eliminarea spațiilor.
 
 
     Atunci când un șir de caractere este eliminat (drop), este eliminată și memoria heap a acestuia.
 
 
-    Șirile de caractere au un operator `+` care extinde șirul cu un `&str`. (Șirul din stânga este `consumat`, iar buffer-ul lui
-    este refolosit pentru a stoca valoarea finală. Șirul din dreapta este doar împrumutat pentru a efectua concatenarea.)
+    Șirile de caractere au un operator `+` care extinde șirul cu un `&str` (șirul din stânga este `consumat`, iar buffer-ul lui
```

```suggestion
    Șirurile de caractere au un operator `+` care extinde șirul cu un `&str` (șirul din stânga este `consumat`, iar buffer-ul lui
```
Typo la scrierea cuvantului cuvantului "sirurile"